app/views/shared/_stats.html.erb

```diff
@@ -0,0 +1,15 @@
+<% @user ||= current_user %>
```

Comment:
??

Reply:
file _stats được sử dụng chung cho page show và page home. Page home hiểm thị thông tin người hiện tại(logged in), page show hiển thị thông tin người dùng mà mình đang xem).Vì vậy nó sẽ tương ứng với cái mình sử dụng ạ

Reply:
pass argument `user` to partial views

Reply:
ý e là khi @user mà nil (tìm không thấy th user) thì nó trả về người dùng đang logged in (ương ứng với home), còn không thì nó giữ nguyên (tương ứng với show)

Reply:
e chưa hiểu ý a lắm ạ, khi direct đến trang home thì không sử dụng @user nên @user= nil => trả về logged in user để hiển thị 
còn direct đến show thì @user sẽ không thay đổi gì, trả về người dùng mình muốn xem, vậy sử dụng thêm biến user thêm để làm gì ạ